JASP-Engine/JASP/R/descriptives.R

```diff
@@ -977,14 +978,24 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 
 .plotMarginal <- function(column, variableName,
-                          rugs = FALSE, displayDensity = FALSE) {
+                          rugs = FALSE, displayDensity = FALSE, binWidthType) {
   column <- as.numeric(column)
   variable <- na.omit(column)
 
   if(length(variable) == 0)
     return(NULL)
+  
+  if(binWidthType == "doane"){
```

Comment:
```suggestion
  if (binWidthType == "doane") {
```
(I know, the if statement above also is inconsistent)